12-offer-encoding.md

```diff
@@ -120,8 +120,8 @@ Each form is signed using one or more *signature TLV elements*: TLV
 types 240 through 1000.  For these,
 the tag is "lightning" || `messagename` || `fieldname`, and `msg` is the
 Merkle-root; "lightning" is the literal 9-byte ASCII string,
-`messagename` is the name of the TLV stream being signed (i.e. "offer", "invoice_request" or "invoice") and the `fieldname` is the TLV field containing the
-signature (e.g. "signature" or "refund_signature").
+`messagename` is the name of the TLV stream being signed (i.e. "invoice_request" or "invoice") and the `fieldname` is the TLV field containing the
+signature (e.g. "signature").
```

Comment:
The example below still uses offer signature. Lines 143 and 174 in 4eb8b86a6429650b151c89914729b52d97d7af77.